packages/sane/lib/src/sane_dev.dart

```diff
@@ -1,19 +1,20 @@
 import 'dart:typed_data';
 
 import 'package:sane/sane.dart';
+import 'package:sane/src/logger.dart';
 
 class SaneDev implements Sane {
   @override
   Future<void> cancel(SaneHandle handle) {
     return Future.delayed(const Duration(seconds: 1), () {
-      print('sane_cancel()');
+      logger.finest('sane_cancel()');
```

Comment:
Can you flag the log with the class name, so we can know that it comes from `SaneDev`?